coral-trino/src/main/java/com/linkedin/coral/trino/rel2trino/CoralToTrinoSqlCallConverter.java

```diff
@@ -135,7 +125,7 @@ protected SqlCall transform(SqlCall sqlCall) {
         new GenericCoralRegistryOperatorRenameSqlCallTransformer(),
 
         new ReturnTypeAdjustmentTransformer(configs), new UnnestOperatorTransformer(), new AsOperatorTransformer(),
-        new JoinSqlCallTransformer(), new NullOrderingTransformer());
+        new JoinSqlCallTransformer(), new NullOrderingTransformer(), new SubstrIndexTransformer());
```

Comment:
Could you make sure every transformer is added in a new line?

Reply:
`spotlessApply` forces this formatting